### PR TITLE
RSpec testing for enumerable methods

### PR DESCRIPTION
**Pull request by jubaan**
RSpec testing for enumerable methods. my_each, my_any?, my_all?, my_none?, my_inject, my_map, my_select and my_count tested for true and false values.

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
## Hi team!
### Nice job! Project looks! :+1: 

### But your project needs a few changes before it can be approved.


#### General

*  Pull Request should has meaningful title and a short description screenshot from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
*  Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. screenshot from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)


#### Section specific
screen
* Make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide

When you cancel changes in .rybocop.yml file kindly recommended running the command: rubocop

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*
*Happy coding *

**Comment by jubaan** [comment]
> ## Hi team!
> ### Nice job! Project looks! +1
> ### But your project needs a few changes before it can be approved.
> #### General
> 
>     * Pull Request should has meaningful title and a short description screenshot from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
> 
>     * Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. screenshot from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/)
> 
> 
> #### Section specific
> 
> screen
> 
>     * Make sure that code is clean: indentation with two spaces, line lengths, spaces around operators - first section of [github/rubocop-github:STYLEGUIDE.md@`master`#ruby-style-guide](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md#ruby-style-guide) [](/github/rubocop-github/blob/HEAD@{2020-02-21T22:03:08Z}/STYLEGUIDE.md#ruby-style-guide)
> 
> 
> When you cancel changes in .rybocop.yml file kindly recommended running the command: rubocop
> 
> _If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you._
> *Happy coding *


Changes:

- Change Pull request name for a meaninful one and short description.
- Restore .rubocop.yml file
- Fix all style linter errors

**Review by fatymahmed — CHANGES_REQUESTED** [review]
## Hi @jubaan  @didymus707,
Great job making some of the changes requested by the previous reviewer 👏👏.
- [x] Please don't change the Linter config files, make sure they are similar to the ones provided by microverse, check [here](https://github.com/microverseinc/linters-config/tree/master/ruby).
- [x] Some of your tests are failing, kindly fix this.
See screenshot of failing tests

### Status: Changes Required 🛑
_Feel free to reach out to me on slack (Fatima) if you have any questions, I'd be happy to help_

File: .rubocop.yml [hunk 1]
```diff
@@ -3,7 +3,8 @@ AllCops:
     - "README.md"
     - "Guardfile"
     - "Rakefile"
-    - --force-exclusion "Methods.rb"
+    - methods.rb
+    - ./spec/spec_helper.rb
```
Comment (fatymahmed) [inline .rubocop.yml]: - Don't change these files

Comment (jubaan) [reply .rubocop.yml]: all of the linter error comes from this files.

**Review by whiz25 — APPROVED** [review]
## Congz :clap: 
You have made all the requested changes and methods cover all edge cases. You can merge and proceed.

### Martin Afani

**Pull request merged**